.github/workflows/matrix.js

```diff
@@ -65,6 +90,36 @@ matrix.addAxis({
       distribution: 'jdkfile',
       url: 'https://download.bell-sw.com/java/11.0.13+8/bellsoft-jdk11.0.13+8-linux-amd64.tar.gz'
     },
+    // SapMachine
+    {
+      group: 'SapMachine',
+      version: '11',
+      jit: 'hotspot',
+      distribution: 'jdkfile',
+      url: 'https://github.com/SAP/SapMachine/releases/download/sapmachine-11.0.17/sapmachine-jdk-11.0.17_linux-x64_bin.tar.gz'
+    },
+    {
+      group: 'SapMachine',
+      version: '17',
+      jit: 'hotspot',
+      distribution: 'jdkfile',
+      url: 'https://github.com/SAP/SapMachine/releases/download/sapmachine-17.0.5/sapmachine-jdk-17.0.5_linux-x64_bin.tar.gz'
+    },
+    //Semeru
+    {
+      group: 'Semeru',
```

Comment:
@juherr - Do you think we should add `Semeru` ? This causes compilation errors in `SimpleBaseTest.kt` which attempts at using `com.sun.management.HotSpotDiagnosticMXBean#dumpHeap` This method doesn't seem to exist in this JDK flavor.  Please see the failure here: https://github.com/krmahadevan/testng/actions/runs/3463174554/jobs/5783049056

Reply:
@juherr This is the same issue that we had with `OpenJ9` :(

Reply:
If possible we should remove the strong dependency.
Maybe the fun could be moved in another class and excluded when not build on hotspot (at tests using this fun too)

Reply:
I will try to figure out what can be done on that and raise a new PR to address it